src/apps/loyalty/server/index.tsx

```diff
@@ -69,6 +70,18 @@ app.get("/inquiries", (req, res) => {
     return res.redirect(req.baseUrl + "/login")
   }
 
+  const headers = { "X-Access-Token": req.user.get("accessToken") }
```

Comment:
That actually reminded me to ask you, i don't think we really need that file at all. The only thing `artsy-passport` expects is a `Backbone.Model` so that it can `set`/`get` attributes.

There's no Backbone fetching actually happening here. In fact Backbone as-is can't run on the server since it uses `ajax`, you'd need to use https://github.com/artsy/backbone-super-sync to get Isomorphic Backbone.

So that's why I decided to skip Backbone entirely. And I'm pretty sure that the `CurrentUser` model can just be `export default class CurrentUser extends Backbone.Model {}`. We don't need any of that other stuff (it won't work server-side).